dftimewolf/lib/collectors/gcloud.py

```diff
@@ -184,5 +206,8 @@ def SetUp(self,
             'disk name?')
```

Comment:
It would be helpful to have the name of the resource you're trying to retrieve here

Reply:
The full detailed exception is now also added in the output.